librosa/effects.py

```diff
@@ -757,6 +757,7 @@ def deemphasis(
 ) -> Tuple[np.ndarray, np.ndarray]:
     ...
 
+
```

Comment:
I don't understand why this one is failing:

```
librosa/effects.py:740: error: Overloaded function signatures 1 and 2 overlap with
incompatible return types  [misc]
    def deemphasis(
```
Why does mypy think that the overloaded signatures overlap?

Reply:
It appears that we're running aground of this: https://github.com/python/mypy/issues/6580

Reply:
Thread linked above provides a (very ugly) solution - we need to drop the default placeholders `= ...` on literals in overloads.  (Or, at least, overloads that do not match the full declaration.)  For example:

```python
@overload
def deemphasis(
    y: np.ndarray,
    *,
    coef: float = ...,
    zi: Optional[ArrayLike] = ...,
    return_zf: Literal[True],
) -> Tuple[np.ndarray, np.ndarray]:
    ...
```

omits the `= ...` on `return_zf`, and thusly validates.  I'm not alone in finding this distasteful, but I infer from the upstream issue thread that there are probably a lot of subtleties to the bigger picture that explain why this is necessary.

Probably this solution will make the vscode tab-complete-coders unhappy, but I don't see another way around it.  It's likely that similar hacks will be necessary in the bulk of `core.convert` where we have overloads everywhere.